webroot/src/components/PrivilegeCard/PrivilegeCard.ts

```diff
@@ -0,0 +1,96 @@
+//
+//  PrivilegeCard.ts
+//  CompactConnect
+//
+//  Created by InspiringApps on 10/8/2024.
+//
+import {
+    Component,
+    Vue,
+    toNative,
+    Prop
+} from 'vue-facing-decorator';
+import { License, LicenseStatus } from '@/models/License/License.model';
+import { State } from '@/models/State/State.model';
+import moment from 'moment';
+
+@Component({
+    name: 'PrivilegeCard',
+})
+class PrivilegeCard extends Vue {
+    @Prop({ required: true }) privilege?: License;
+
+    //
+    // Computed
+    //
+    get statusDisplay():string {
```

Comment:
same here on spacing? 
```suggestion
    get statusDisplay(): string {
```